kDrive/UI/Controller/Menu/AppLockSettingsViewController.swift

```diff
@@ -18,6 +18,7 @@
 
 import kDriveResources
 import LocalAuthentication
+import kDriveCore
```

Comment:
Why do we now need core ?

Reply:
`MatomoUtils` in now in core. (as needed by the player)